article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -254,14 +255,35 @@ object DotcomponentsDataModel {
 
     val bodyBlocks: List[Block] = {
       val bodyBlocks = blocks.body.getOrElse(Nil)
-      bodyBlocks.map(block => Block(block.bodyHtml, blocksToPageElements(block.elements, shouldAddAffiliateLinks))).toList
+        .map(block => Block(block.bodyHtml, blocksToPageElements(block.elements, shouldAddAffiliateLinks)))
+        .toList
+
+      val last60 = blocks.requestedBodyBlocks
+        .getOrElse(Map.empty[String, Seq[APIBlock]])
+        .getOrElse("body:latest:60", Seq.empty[APIBlock])
+        .map(block => Block(block.bodyHtml, blocksToPageElements(block.elements, shouldAddAffiliateLinks)))
+        .toList
+
+      // This is the liveblog case
```

Comment:
Not entirely sure how I feel about branching for the liveblog case here, as it's pretty buried in the larger method.

Reply:
I think it's ok. It's all contained in the body of the definition of `val bodyBlocks: List[Block]`, so makes sense.